app/src/androidTest/java/org/mozilla/focus/activity/EnhancedTrackingProtectionSettingsTest.kt

```diff
@@ -117,6 +117,7 @@ class EnhancedTrackingProtectionSettingsTest {
         }
     }
 
+    @Ignore
```

Comment:
Should we file tickets for these UI tests that we are ignoring ? :)

Reply:
🙌🏽

Reply:
Added